AK/Vector.h

```diff
@@ -86,22 +86,26 @@ requires(!IsRvalueReference<T>) class Vector {
         other.reset_capacity();
     }
 
+#ifdef KERNEL
+    Vector(Vector const&) = delete;
+
+    template<size_t other_inline_capacity>
+    Vector(Vector<T, other_inline_capacity> const&) = delete;
```

Comment:
Note for the future:
For constexpr Vector, we should look into if making this available `consteval` in the Kernel is a) worth it and b) still has nice errors